### PR TITLE
updated settings so that there is no decrepation warning

### PR DESCRIPTION
**Pull request by maxxst**
I got warnings about `TEMPLATE_DEBUG = True` and that it's [decrepated](https://docs.djangoproject.com/en/1.8/ref/settings/#template-debug). So I changed it the way It's supposed to be ... I think

### CODE REVIEW

File: wagtail/project_template/project_name/settings/production.py [hunk 1]
```diff
@@ -2,7 +2,10 @@
 
 
 DEBUG = False
-TEMPLATE_DEBUG = False
+DEBUG_TEMPLATE = False
```
Comment (kaedroho) [inline wagtail/project_template/project_name/settings/production.py]: We could probably leave this one out

File: wagtail/project_template/project_name/settings/dev.py [hunk 2]
```diff
@@ -3,7 +3,10 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
-TEMPLATE_DEBUG = True
+DEBUG_TEMPLATE = True
+
+for i in range(0, len(TEMPLATES) - 1):
```
Comment (kaedroho) [inline wagtail/project_template/project_name/settings/dev.py]: Could be changed to:
```
for template_engine in TEMPLATES:
    template_engine['OPTIONS']['debug'] = True
```

looks a bit nicer I think

**Comment by kaedroho** [comment]
@maxxst Thanks for spotting and fixing. I've made a couple of little suggestions above

**Comment by maxxst** [comment]
So I hope that's what you meant :)

**Comment by kaedroho** [comment]
Thanks!

**Pull request merged**